playground/waitfor/WaitForSandbox.AppHost/Program.cs

```diff
@@ -7,7 +7,15 @@
                 .WithPasswordAuthentication()
                 .RunAsContainer(c =>
                 {
-                    c.WithPgAdmin();
+                    c.WithPgAdmin(c =>
+                    {
+                        c.WithHostPort(15551);
+
+                        // Variables that we are going to need to set when we detect that
+                        // we are run run mode with codespaces.
+                        c.WithEnvironment("PGADMIN_CONFIG_PROXY_X_HOST_COUNT", "1");
+                        c.WithEnvironment("PGADMIN_CONFIG_PROXY_X_PREFIX_COUNT", "1");
```

Comment:
To be discussed. We may want to provide a mechanism for extensions to query whether they are working in codespaces and if so inject some additional configuration values. This is required for PGAdmin. This applies to Codespaces only - for local container dev and probably remote container dev (over SSH) this is not an issue.